examples/gno.land/r/demo/microblog/microblog.gno

```diff
@@ -0,0 +1,51 @@
+// microblog is a website with shortform posts from users.
+// the API is exteremly simple - "AddPost" takes markdown and
+// adds it to the users site. The user is determined by
+// the address and your microblog lives at
+// /r/demo/microblog:<YOUR-ADDRESS>
+//
+// Getting started:
+//
+// (One-time) Add the microblog package:
+//
+//	gnokey maketx addpkg --pkgpath "gno.land/p/demo/microblog" --pkgdir "examples/gno.land/p/demo/microblog" \
+//		--deposit 100000000ugnot --gas-fee 1000000ugnot --gas-wanted 2000000 --broadcast --chainid dev --remote localhost:26657 <YOURKEY>
+//
+// (One-time) Add the microblog realm:
+//
+//	gnokey maketx addpkg --pkgpath "gno.land/r/demo/microblog" --pkgdir "examples/gno.land/r/demo/microblog" \
+//		--deposit 100000000ugnot --gas-fee 1000000ugnot --gas-wanted 2000000 --broadcast --chainid dev --remote localhost:26657 <YOURKEY>
+//
+// Add a microblog post:
+//
+//	gnokey maketx call --pkgpath "gno.land/r/demo/microblog" --func "NewPost" --args "hello, world" \
+//		--gas-fee "1000000ugnot" --gas-wanted "2000000" --broadcast --chainid dev --remote localhost:26657 zkey
```

Comment:
This tutorial should live outside of the realm code, because we tend to change these commands / flags, and it will quickly become outdated. What do you think about adding a README in the realm directory?


I'd love to discuss more on this topic of having realm / package tutorials bundled with the gno code itself somehow (not in-code, but probably in an accompanying README) cc @moul @ajnavarro @thehowl

Reply:
:+1: to add a README in the directory.

Reply:
I'm a fan of README's in the directory. 

Just brainstorming here, but it might be interesting to have Gno render doc comments specially (alongside the code).